docs/src/templates/pages/documentation/js/index.hbs

```diff
@@ -1,85 +1,79 @@
+Thanks to these conventions it has become very easy for engineers to pick up a new codebase and get up and running quickly, as well as improving the onboarding process for new starters.
 
-Once the polyfills have loaded, we then run the rest of our bundled JavaScript and load in the JUST EAT legacy script files (these are files that we will convert over to our new module syntax as we get chance).
+Any discussions are documented through Github issues and Pull Requests, and then when put into practice, should be documented in the relevant section of the fozzie documentation. All UI engineers are encouraged to feed back in to this process as we see our code conventions as an ever evolving discussion rather than permanently set in stone.
 
+For details on our agreed JavaScript coding conventions see the [Code Style section]({{ ../baseUrl }}/documentation/js/codestyle).
 
-<hr class="sectionSplitter">
 
-<a name="libs"></a>
-## Libraries
+<h3 class="sg-sectionHeading">Unit Testing</h3>
 
-If you need to add any 3rd party library code, it should be placed in the [assets/src/js/libs/](https://github.com/trykickoff/kickoff/tree/master/assets/src/js/libs) folder.
+All JavaScript code should include unit tests. We use the [Jest framework](https://facebook.github.io/jest/).
 
-Our aim is to keep our JavaScript as lightweight as possible.  Therefore, before adding a library, think about the value it is giving the project and weigh it up against it’s filesize footprint.
+See the [JavaScript unit testing section](#todo) for details on best practices and recommended coding styles.
 
-Currently, we use the following libraries:
 
----
+<a name="js-babel"></a><h2 class="sg-sectionHeading">Babel</h2>
 
-<a name="js-modernizr"></a>
-### Modernizr
+We use Babel which allows us to write code using ES2015 syntax and then transform it back to ES5. This means that it is compatible with a wider variety of browsers.
 
-We use a custom build of Modernizr configured to detect some commonly used CSS features.
+To get an idea of what the transformed code will look like you can try out the [Babel REPL](https://babeljs.io/repl/).
 
-You may wish to extend this [custom Modernizr build](http://www.modernizr.com/download/) and replace the current version.  To do this, ensure you copy the url in the current [modernizr.min.js](https://github.je-labs.com/I18N-Production-ROTW/ConsumerWeb/blob/master/src/ConsumerWeb/Assets/src/js/libs/modernizr.min.js) so that the current tests are preserved in your new custom build.
+This transformation happens as part of the [`gulp scripts`](https://github.com/justeat/gulp-build-fozzie#scripts) build task – a task that is provided by the [‘gulp-build-fozzie’ NPM module](https://github.com/justeat/gulp-build-fozzie).
```

Comment:
Fun fact — Babel have actually changed their terminology to say compiler rather than transpiler recently.

Reply:
Nope – wasn't really trying to avoid. I think as long as you guys understand what's being conveyed though, it's probably ok as is (as it could be transformed/transpiled/compiled – take your pick really!)